src/main/java/com/lob/net/ResponseGetter.java

```diff
@@ -124,6 +124,20 @@ public Parameter(String key, Object value) {
         return conn;
     }
 
+    private static java.net.HttpURLConnection createJsonPostConnection(String url, String data, String query, RequestOptions options) throws IOException {
```

Comment:
@amaan-lob how come this needed to be added did the existing NORMAL request type not do it?

Reply:
The NORMAL request type sets the content-type header to `application/x-www-form-urlencoded` whereas we need `application/json` for these endpoints.